lib/src/common/disposable.dart

```diff
@@ -284,6 +300,14 @@ class Disposable implements _Disposable, DisposableManagerV3 {
       _logger.info(
           '$runtimeType $hashCode took ${stopwatch.elapsedMicroseconds / 1000000.0} seconds to dispose');
     }
+
+    flagLeak(runtimeType.toString());
```

Comment:
Derp, never mind. I assumed leakFlag was meant to provide consumers a way to add a more explicit description that would be used to describe the leak flag.